debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -520,11 +519,7 @@ private short getDefaultTopicReplicationFactor(AdminClient admin) throws Excepti
                 return Short.parseShort(defaultReplicationFactorValue);
             }
         }
-        catch (ExecutionException ex) {
-            // ignore UnsupportedVersionException, e.g. due to older broker version
-            if (!(ex.getCause() instanceof UnsupportedVersionException)) {
-                throw ex;
-            }
+        catch (Exception ex) {
```

Comment:
Hum, but we cannot simply swallow every kind of exception. Why is it you don't assign the right privileges?